src/app/components/layout/side-menu/side-menu.component.css

```diff
@@ -1,11 +1,11 @@
 .menu-list {
   background-color: white !important;
-  min-height: 100vh;
-  margin-top: 0.25%;
+  width: 100%; 
```

Comment:
Why is this required?

Reply:
this makes slightly change in width. But not very much required. I'll remove it.